checkstyle.xml

```diff
@@ -178,7 +178,7 @@
     <module name="FinalClass"/>
     <module name="HideUtilityClassConstructor"/>
     <module name="InterfaceIsType"/>
-    <module name="VisibilityModifier"/>
+    <!-- <module name="VisibilityModifier"/> -->
```

Comment:
I wouldn't left any "code" commented out, I'd rather delete it.

Reply:
I can delete these, but it has been a pattern in the checkstyle configuration cause it's impossible to remember what it supports and doesn't and (re)add things in this ridiculous XML. LMK if you still want me to remove all the commented out options?

Reply:
I'd prefer removing as I'd still have to Google the option when I want to add a it back. I guess it makes sense when you create a file, but once you have a working configuration I'd remove all the comments, kind of what you do with http servers configurations.